<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarsDrilling</class>
 <widget class="QMainWindow" name="MarsDrilling">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarsDrilling</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Динамический процесс бурения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Вид сверху</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Вид сбоку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Помощь</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>example.ascii</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
